src/keys-builder/typescript/marker.extractor.ts

```diff
@@ -1,6 +1,6 @@
 import { SourceFile } from 'typescript';
 import { tsquery } from '@phenomnomnominal/tsquery';
-import * as ts from 'typescript';
+import { Node } from 'typescript';
```

Comment:
Merge the `typescript` imports, check if there are other places where we might have duplications

Reply:
I think it's good to distinguish typing imports from guard imports, but I'm fine with that.

Reply:
Sorry, I'm too used to the linter doing that for me.

Reply:
@kekel87 You are welcome to add it as part of the linter here 😄

Reply:
I could do it, but it would be in another PR.